minion/globals.cpp

```diff
@@ -37,6 +37,6 @@ Globals::~Globals() {
   delete this->state_m;
   delete this->queues_m;
   delete this->options_m;
-  delete this->parData_m;
+  //delete this->parData_m; //FIXME: having this in causes runMinion to segfault!
```

Comment:
On my Macbook, when running my example, this segfaults and I don't know why....

Reply:
![Screenshot 2023-10-19 at 17 59 28](https://github.com/minion/minion/assets/44070547/b339a952-8ee1-4d90-8e9b-00ae1645c65e)